src/Core/src/ImageSources/StreamImageSourceService/StreamImageSourceService.Android.cs

```diff
@@ -21,7 +21,7 @@ public partial class StreamImageSourceService
 				Stream? stream = null;
 				try
 				{
-					stream = await streamImageSource.GetStreamAsync(cancellationToken).ConfigureAwait(false);
+					stream = await streamImageSource.GetStreamAsync(cancellationToken);
```

Comment:
Do we still need this?

Reply:
Yes. This is the actual fix.

Reply:
Nope, the post() fix by itself does not fix this. In fact, this change by itself works even without the post() change.